ui/settings_page.go

```diff
@@ -54,14 +60,15 @@ func (win *Window) SettingsPage(common pageCommon) layout.Widget {
 		beepNewBlocks:    new(widget.Bool),
 		connectToPeer:    new(widget.Bool),
 		userAgent:        new(widget.Bool),
+		chevronRightIcon: chevronRightIcon,
 
 		line:     common.theme.Line(),
 		errChann: common.errorChannels[PageSettings],
 
-		currencyConversion:  common.theme.PlainIconButton(new(widget.Clickable), icon),
-		updateConnectToPeer: common.theme.PlainIconButton(new(widget.Clickable), icon),
-		updateUserAgent:     common.theme.PlainIconButton(new(widget.Clickable), icon),
-		changeStartupPass:   common.theme.PlainIconButton(new(widget.Clickable), icon),
+		currencyConversion:  currencyConversion,
+		updateConnectToPeer: updateConnectToPeer,
+		updateUserAgent:     updateUserAgent,
```

Comment:
You don't need to create variables for the Clickable types, you can just use new(widget.Cickable) here.